ta_services/keymaster/km.h

```diff
@@ -5,7 +5,10 @@
 #define LOCAL_KM_H
 
 #include <tee_internal_api.h>
+#include <keymaster_ta.h>
 
+bool km_is_configured(uint32_t cmd);
```

Comment:
The `km_` class of function doesn't use `cmd`. I guess the functions that care whether KM is configured or not will call this function, and the rest not, so the `cmd` parameter is not needed.

Reply:
https://source.android.com/security/keystore/implementer-ref#configure says:

> Until this method is called, all other methods return KM_ERROR_KEYMASTER_NOT_CONFIGURED.

Do you mean for all other functions in `km.c` to call this function rather than just calling it once in `entry.c`?